### ISSUE TITLE
Issue with InverseCrosstile3D

### ISSUE THREAD

**Issue by jzwar** [tataratat/splinepy#188]
The edge cases seem to be redundant and result in tangled splines. I don't know where they came from but we might need to check their consistency!

Concerns all splines 12-end in the create_tile-function

**Comment by j042** [comment]
If this is a new issue since 0.0.26, maybe you can check whether cartesian_product is used correctly. That's something I changed during the migration

**Comment by jzwar** [comment]
No, I think there is a definition problem. I will need to look into that once I am back in the Institute.
